### PR TITLE
EREGCSC-921: Adjust styling of supplemental content subcategories

### PR DESCRIPTION
**Pull request by PhilR8**

### CODE REVIEW

**Review by knollfear — COMMENTED** [review]
This looks good.  Just 2 small style issues and I can approve it.

File: regulations/static/components/SupplementalContentCategory.vue [hunk 1]
```diff
@@ -59,8 +71,11 @@ export default {
     },
 
     computed: {
-        showDescription: function() {
-            return (this.description && !/^\s*$/.test(this.description));
+        showDescription: function () {
+            return this.description && !/^\s*$/.test(this.description);
+        },
+        has_sub_categories() {
+            return !!this.sub_categories.length;
```
[the remark this answers is not in the document]
Comment (PhilR8) [reply regulations/static/components/SupplementalContentCategory.vue:78]: I'm explicitly coercing the truthy/falsey length into a boolean, which made sense to me at the time I wrote this.  Although I'm just doing a little checking around and opinions differ if that's necessary or i[f it is needlessly more verbose](https://www.oreilly.com/library/view/you-dont-know/9781491905159/ch04.html).  I can edit this to just use length.

File: regulations/static/components/SupplementalContentCategory.vue [hunk 2]
```diff
@@ -5,28 +5,40 @@
                 <template v-slot:expanded>{{ name }} <i class="fa fa-chevron-up"></i></template>
                 <template v-slot:collapsed>{{ name }} <i class="fa fa-chevron-down"></i></template>
             </collapse-button>
-            <span v-if="showDescription" class="category-description">{{ description }}</span>
-            <collapsible :name="name" state="collapsed" class="category-content">
-                <supplemental-content-category v-for="(category, index) in sub_categories" :key="index"
+            <span v-if="showDescription" class="category-description">{{
```
[the remark this answers is not in the document]
Comment (PhilR8) [reply regulations/static/components/SupplementalContentCategory.vue:8]: I'm with you on this, but my Prettier config changes it to how you see it in the PR.  I wonder if there's an override I can add for this specific case.

File: regulations/static/components/SupplementalContentCategory.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (knollfear) [reply regulations/static/components/SupplementalContentCategory.vue:78]: I think length is what I am used to seeing.  I think a more explicit cast like Boolean(this.sub_categories.length) or this.sub_categories.length > 0 would be fine too.

[the remark this answers is not in the document]
Comment (cgodwin1) [reply regulations/static/components/SupplementalContentCategory.vue:78]: I think the "has_" in the name implies truthiness making it not needed from a readability perspective

[the remark this answers is not in the document]
Comment (PhilR8) [reply regulations/static/components/SupplementalContentCategory.vue]: hmm I've never heard of snake case being used to convey truthiness in JS - is that a common convention in another language?

**Review by knollfear — APPROVED** [review]
LGTM

**Pull request merged**